Source/ThirdParty/skia/CMakeLists.txt

```
@@ -86,8 +87,11 @@ if (DEVELOPER_MODE AND (SKIA_ROOT_DIR OR NOT "$ENV{SKIA_ROOT_DIR}" STREQUAL ""))
     target_link_libraries(Skia INTERFACE
         Fontconfig::Fontconfig
         Freetype::Freetype
+        JPEG::JPEG
         OpenGL::GLES
         PNG::PNG
+        WebP::libwebp
+        WebP::mux
```

Isn't it enough to list `WebP::mux`? The imported target should automatically depend on `WebP::libwebp`, which means we would want the following in `FindWebP.cmake`:

```cmake
target_link_libraries(WebP::demux INTERFACE WebP::libwebp)
target_link_libraries(WebP::mux INTERFACE WebP::libwebp)
```

and then here:

```suggestion
        WebP::mux
```

---

I don't know, but I think I prefer to be explicit here, skia doesn't depend only on mux component, but on webp and the mux component. ImageDecoders.cmake also lists both.

---

Using `INTERFACE` when declaring the dependency (as suggested) causes CMake to include the compiler and linker flags for both when using the `WebP::mux` target. There is no ambiguity, adding `WebP::libwebp` explicitly here doesn't add any value. 

The fact that `ImageDecoders.cmake` uses both is a mistake, and only needed because the dependency declaration is presently missing in `FindWebP.cmake`.